src/Stream.php

```diff
@@ -70,15 +70,6 @@ public function __construct($stream, $options = [])
         $this->uri = $this->getMetadata('uri');
     }
 
-    public function __get($name)
```

Comment:
The stream property is unset when detaching. So this throws an error when accessed in some places.